### PR TITLE
Features/optional fnt gnt

### PR DESCRIPTION
**Pull request by ryanbnl**
- `fnt` and `gnt` are now optional - one of the two is required. This is conform the latest regulations.
- Examples and tests updated for these new cases (and for several different sequences).
- merge.py now outputs un-html-encoded text in modern Python.

### CODE REVIEW

File: DCC.Core.Types.schema.json [hunk 1]
```diff
@@ -1,70 +1,73 @@
...
-          "title": "Forename",
-          "description": "The forename(s) of the person addressed in the certificate",
-          "type": "string",
-          "maxLength": 80,
-          "examples": [
-            "Jiřina-Maria Alena"
-          ]
+        "person_name": {
+            "description": "Person name: Surname(s), forename(s) - in that order",
+            "anyOf": [{
```
Comment (dslmeinte) [inline DCC.Core.Types.schema.json:19]: This is not formatted, and the `make` execution complains about it.

**Review by dslmeinte — APPROVED** [review]

**Comment by skounis** [comment]
@ryanbnl I wasn't able to locate the updated regulation document. Do you have it handy?

PR looks fine

**Review by skounis — APPROVED** [review]

**Comment by lari-kannisto** [comment]
The Implementing Decicion was updated as part of the DCC revocation  https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32022D0483
The changes are in ANNEX II chapter 3.2.

**Comment by dslmeinte** [comment]
@skounis @ryanbnl @lari-kannisto This change was discussed in last Friday's (June 10th) eHN TechIOP meeting. It should go up for proposal to the Coordinated Actions meeting on this Wednesday (June 15th). It should therefore be merged to `main` so we can ready a release, right?

**Pull request merged**